ui/app/serializers/aws/root-config.js

```diff
@@ -10,13 +10,6 @@ export default class AwsRootConfigSerializer extends ApplicationSerializer {
     if (!payload.data) {
       return super.normalizeResponse(...arguments);
     }
-    // remove identityTokenTtl and maxRetries if the API's default value of 0 or -1, respectively. We don't want to display this value on configuration details if they haven't changed the default value
```

Comment:
this makes sense to me to remove, any response manipulation for read views is a slippery slope!